invokeai/backend/model_manager/probe.py

```diff
@@ -643,17 +646,19 @@ def _guess_name(self) -> str:
         return name
 
 
-class TextualInversionFolderProbe(FolderProbeBase):
-    def get_format(self) -> ModelFormat:
-        return ModelFormat.EmbeddingFolder
-
-    def get_base_type(self) -> BaseModelType:
-        path = self.model_path / "learned_embeds.bin"
-        if not path.exists():
+class TextualInversionFolderProbe(TextualInversionCheckpointProbe):
```

Comment:
I don't like how this results in a "folder" model having a path that points to a file...

Reply:
Yeah, I'm not really sure what value of storing it as a folder even are for these though. I haven't found any lora or embeddings that have other contextual files that affect generations. Just seems to put us in a position where we have to enforce folder structure for no reason to me.

Reply:
The folder was introduced by hugging face. TIs usually contain two files--one with the weights and another with the name of the concept. 

For Lora's the folder is used to give the model a name